File: devops-scm-api/src/main/java/com/tencent/devops/scm/api/pojo/Change.java
package com.tencent.devops.scm.api.pojo;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Change {
    private String path;
    private Boolean added;
    private Boolean renamed;
    private Boolean deleted;
    private String sha;
    private String blobId;
    private String oldPath;
}

File: devops-scm-api/src/main/kotlin/com/tencent/devops/scm/api/pojo/Change.kt
package com.tencent.devops.scm.api.pojo

data class Change(
    val path: String,
    val added: Boolean = false,
    val renamed: Boolean = false,
    val deleted: Boolean = false,
    val sha: String,
    val blobId: String,
    val oldPath: String? = null
)
